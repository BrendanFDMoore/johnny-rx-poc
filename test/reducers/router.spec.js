import reducers from 'reducers';
import * as Actions from 'actions';
import { LOCATION_CHANGE } from 'constants/index';

describe('Router', () => {
  it('should return the initial state', () => {
    expect(reducers.router(undefined, {})).toMatchSnapshot();
  });

  it(`should handle ${LOCATION_CHANGE}`, () => {
    expect(reducers.router(undefined, Actions.goTo('/'))).toMatchSnapshot();
  });
});
