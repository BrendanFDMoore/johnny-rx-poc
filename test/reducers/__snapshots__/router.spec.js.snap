exports[`Router should handle @@router/LOCATION_CHANGE 1`] = `
Object {
  "action": "PUSH",
  "location": Object {
    "hash": "",
    "key": undefined,
    "pathname": "/",
    "search": undefined,
    "state": undefined,
  },
}
`;

exports[`Router should return the initial state 1`] = `
Object {
  "action": "POP",
  "location": Object {
    "hash": "",
    "key": undefined,
    "pathname": "/",
    "search": "",
    "state": undefined,
  },
}
`;
